libraries/core-react/src/Chip/Chip.tsx

```diff
@@ -159,14 +170,14 @@ export const Chip = forwardRef(function EdsChips(
       if (deletable) {
         handleDelete(event)
       }
-      // Delete takes presidens, else click action is activated
+      // Delete takes precedence, else click action is activated
       if (clickable && !deletable) {
         handleClick(event)
       }
     }
   }
 
-  const resizedChildren = React.Children.map(children, (child) => {
+  const resizedChildren = React.Children.map(children, (child: React.DetailedReactHTMLElement<any, any>) => {
```

Comment:
This <any> gives quite a few warnings in eslint :)

Reply:
Fixed this by changing to type React.ReactElement